README.md

```diff
@@ -1,2 +1,75 @@
 # hello-nextflow
 Training materials for a Nextflow beginners workshop 2024
+
+## Day 2  
+
+Adapated from
+[nf-training](https://github.com/nextflow-io/training/blob/master/nf-training/script7.nf).  
+
+### Installation (temporary)  
+
+Install mamba and packages:
```

Comment:
This install got the updated versions of nextflow on the VM with recommended docker post-install steps

Install mamba:  

```bash
wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
bash Miniforge3-$(uname)-$(uname -m).sh
# complete prompts with defaults

# confirm installation versions
mamba --version 
```

```console
mamba 1.5.8
conda 24.3.0
```

```bash
mamba list | grep -E "nextflow|salmon|mkdocs |mkdocs-material"
```

```console
mkdocs                    1.6.0              pyhd8ed1ab_0    conda-forge
mkdocs-material           9.5.31             pyhd8ed1ab_0    conda-forge
mkdocs-material-extensions 1.3.1              pyhd8ed1ab_0    conda-forge
nextflow                  24.04.4              hdfd78af_0    bioconda
salmon                    1.10.3               h6dccd9a_2    bioconda
```

Install docker:
```bash
# Add Docker's official GPG key:
sudo apt update
sudo apt install ca-certificates curl
sudo install -m 0755 -d /etc/apt/keyrings
sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
sudo chmod a+r /etc/apt/keyrings/docker.asc

# Add the repository to Apt sources:
echo \
  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
sudo apt-get update

# Install the latest Docker packages
sudo apt install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin

# Check installed correctly
sudo docker run hello-world

# post-install for permissions
sudo groupadd docker
sudo usermod -aG docker $USER
newgrp docker
docker run hello-world # confirm
```